server/mdm/android/tests/enterprise/enterprise_test.go

```diff
@@ -50,3 +62,40 @@ func (s *enterpriseTestSuite) TestGetEnterprise() {
 	s.Do("DELETE", "/api/v1/fleet/android_enterprise", nil, http.StatusOK)
 	s.DoJSON("GET", "/api/v1/fleet/android_enterprise", nil, http.StatusNotFound, &resp)
 }
+
+func (s *enterpriseTestSuite) TestEnterpriseSSE() {
+	s.SetupTest()
+
+	// Test happy path
+	resp := s.Do("GET", "/api/v1/fleet/android_enterprise/signup_sse", nil, http.StatusOK)
+	sseDone := make(chan struct{})
+	buf := make([]byte, 1024)
+	go func() {
+		n, _ := resp.Body.Read(buf)
```

Comment:
Assert error instead of ignoring.

Reply:
Not sure if there was a specific reason to have the 1024 bytes buffer, but you could use `io.ReadAll(resp.Body)` to avoid needing to pre-declare/pre-allocate a buf.